docs/webpack.config.ts

```diff
@@ -51,12 +52,18 @@ const webpackConfig: webpack.Configuration = {
     //   'process.env.CLIENT_ID': `'${process.env.CLIENT_ID}'`,
     //   'process.env.CLIENT_SECRET': `'${process.env.CLIENT_SECRET}'`,
     // }),
+    new CleanWebpackPlugin(),
```

Comment:
Can we use [`output.clean`](https://webpack.js.org/guides/output-management/#cleaning-up-the-dist-folder) instead?

Reply:
`output.clean` is Webpack v5+ only.

Reply:
Ah, I missed we were still on Webpack 4.